src/components/task-cards/components/task-card/TaskCard.tsx

```diff
@@ -86,15 +86,16 @@ export default function TaskCard({
 
   return (
     <div
-      className={clsx(classes.card, data.task_details_id ? classes.pointerCard : null)}
+      // className={clsx(classes.card, data.task_details_id ? classes.pointerCard : null)}
+      className={classes.card}
       onMouseLeave={hideActionHandler}
-      onClick={() => {
-        if (data.task_details_id) {
-          navigate(`/${data.module_reference}/${data.task_details_id}`, {
-            state: {source: location.pathname},
-          })
-        }
-      }}
+      // onClick={() => {
```

Comment:
It was requested by Design team, onClick on card should also work

Reply:
all modules use cases weren't handled here. it was leading to broken pages. need to come up with reusable utility. had to comment out temporarily for prod.
@sohhamm

Reply:
@sohhamm fixed actions on card click